src/diffusers/models/attention.py

```diff
@@ -539,3 +548,33 @@ def forward(self, x, emb):
         x = F.group_norm(x, self.num_groups, eps=self.eps)
         x = x * (1 + scale) + shift
         return x
+
+
+class SpatialNorm(nn.Module):
```

Comment:
Could we add a link to where SpatialNorm comes from (is this defined in some paper? :-))

Reply:
Was introduced in the MOVQ paper, added, thanks!